inst/include/cpp11/r_vector.hpp

```diff
@@ -302,8 +350,6 @@ class r_vector : public cpp11::r_vector<T> {
 
   explicit r_vector(const R_xlen_t size);
 
-  ~r_vector();
```

Comment:
No longer need a destructor, which was just freeing the writable version of `protect_`.

The read only parent destructor is going to be called to free its version of `protect_` (which is now the only one)